### PR TITLE
Adding ability to change default model path

### PR DESCRIPTION
**Pull request by Keeo**
Infinity model expects that user will set it as model on controller. I believe this should be handled internally or at least settable. I don't like the code, but can't figure out way how to make it cleaner. If you would like to merge it anyway, give me echo and I'll update readme file.

### CODE REVIEW

**Comment by hhff** [comment]
Naw this is great!  Update readme & squash to a simple commit & I'll merge :)

**Comment by Keeo** [comment]
done

File: README.md [hunk 1]
```diff
@@ -21,8 +21,17 @@ import InfinityRoute from "ember-infinity/mixins/route";
...
 export default Ember.Route.extend(InfinityRoute, {
   model() {
+    var args = {
+      // Load 12 records in one group.
+      perPage: 12,
+      // Starting from page 1.
+      startingPage: 1,
+      /* Access model on 'controller.model'. This is required only if you place your model
+         on different location (when overriding setupController method for example). */
+      modelPath: 'controller.model'
```
Comment (hhff) [inline README.md]: I like how you've lay'd this out, but can we plz move this to the advance usage section?

**Comment by Keeo** [comment]
Would you like this instead?

### ModelPath

ModelPath is optional parameter for situations when you are overriding *setupController*
or when your model is on different location than *controller.model*.
```js
model: function() {
  return this.infinityModel("product", {modelPath: 'controller.products'});
}
setupController: function(controller, model) {
  controller.set('products', model);
}
```

**Comment by hhff** [comment]
I think here would make the most sense:




### infinityModel

You can also provide additional parameters to `infinityModel` that
will be passed to your backend server in addition to the
pagination params. For instance, in the following example a `category`
parameter is added:

```js
return this.infinityModel("product", { perPage: 12, startingPage: 1,
                                       category: "furniture" });
```

* **modelPath**

`modelPath` is optional parameter for situations when you are overriding *setupController*
or when your model is on different location than *controller.model*.
```js
model: function() {
  return this.infinityModel("product", {
    perPage: 12,
    startingPage: 1,
    modelPath: 'controller.products'
  });
},
setupController: function(controller, model) {
  controller.set('products', model);
}
```

### infinity-loader

**Comment by Keeo** [comment]
done

**Comment by hhff** [comment]
u da man!

**Pull request merged**